daemons/gptp/common/ptp_message.cpp

```diff
@@ -1364,18 +1366,19 @@ void PTPMessagePathDelayRespFollowUp::processMessage(IEEE1588Port * port)
 	 * than a certain number of lost FUPs we should set asCapable
 	 * to false
 	 */
+	diffLastSeqId = sequenceId - port->getLastSeqId();
```

Comment:
I have some concerns about this calculation, and possibly the comparison that follows.  I suspect it may run afoul of the integer promotion and usual arithmetic conversion rules.